hazelcast/src/main/java/com/hazelcast/jet/impl/util/LoggingUtil.java

```diff
@@ -50,6 +50,12 @@ public static void logFine(ILogger logger, String template, Object arg1, Object
         }
     }
 
+    public static void logFinest(ILogger logger, String message) {
+        if (logger.isFinestEnabled()) {
```

Comment:
This is pointless. You call `isFinestEnabled` explicitly if you want to avoid doing extra work if it's disabled. `isFinestEnabled` is called as the first thing inside of `logger.finest`.

Reply:
done